ui/toolbar.ini

```diff
@@ -36,8 +39,13 @@ name = Portrait
 name[de] = Hochformat
 name[it] = Ritratto
 name[ru] = Портретная
+<<<<<<< HEAD
```

Comment:
Seems my rebasing skills are still lacking :man_facepalming: I'm pretty sure this was the culprit. 
@ja-he: I'll test it out, just to be sure, then I'll update the PR.